framework/source/class/qx/ui/window/Window.js

```diff
@@ -929,6 +1055,40 @@ qx.Class.define("qx.ui.window.Window",
       }
     },
 
+    // overridden
```

Comment:
apply functions are always overridden. They override the default code. Or at least that's the way it is throughout most qooxdoo code. I don't care. I'll remove that, but I think it's "the standard."